atlas-sdk/build.gradle.kts

```diff
@@ -48,6 +51,8 @@ dependencies {
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
     implementation("com.google.code.gson:gson:2.10.1")
 
+    implementation("androidx.constraintlayout:constraintlayout:2.2.0")
```

Comment:
Our UI consists of a single WebView. I'd strongly argue against any UI library, even the smallest one, in the SDK package.